src/themes/app.css

```diff
@@ -135,7 +135,11 @@ p {
 }
 
 a {
-    color: inherit;
+    color: #615bc2;
```

Comment:
Can we avoid this? Is `components.MuiLink.styleOverrides.root` not working?

Reply:
The problem is that not every "link" uses a MUI Link component. It seems like we have a lot of plain anchor tags all over the app, so this has a lot more coverage.